Source/WebCore/rendering/svg/legacy/LegacyRenderSVGContainer.h

```diff
@@ -73,8 +74,8 @@ class LegacyRenderSVGContainer : public LegacyRenderSVGModelObject {
     bool isLegacySVGContainer() const final { return true; }
 
     FloatRect m_objectBoundingBox;
-    FloatRect m_strokeBoundingBox;
     FloatRect m_repaintBoundingBox;
+    mutable FloatRect m_strokeBoundingBox { FloatRect::nanRect() };
```

Comment:
Seems like a good place to use `Markable<>`

Reply:
Sounds good, changed.